test/custom-formatter-test.cc

```diff
@@ -54,3 +54,29 @@ TEST(CustomFormatterTest, Format) {
   EXPECT_EQ("0.00", custom_format("{:.2f}", -.00001));
 }
 #endif
+
+#ifdef FMT_USE_STRING_VIEW
+struct explicitly_convertible_to_std_string_view {
+  explicit operator std::string_view() const { return "foo"; }
+};
+
+namespace fmt {
+
+template <>
+class formatter<explicitly_convertible_to_std_string_view>
+    : public formatter<std::string_view> {
+ public:
+  template <typename format_context>
+  auto format(const explicitly_convertible_to_std_string_view& v,
+              format_context& ctx) {
+    return format_to(ctx.out(), "'{}'", static_cast<std::string_view>(v));
+  }
+};
+
+}  // namespace fmt
+
+TEST(CustomFormatterTest, FormatExplicitlyConvertibleToStdStringView) {
+  EXPECT_EQ("'foo'",
+            fmt::format("{}", explicitly_convertible_to_std_string_view()));
+}
+#endif
```

Comment:
Please move the test to `format-test.cc` since `custom-formatter` tests a different API.